netlify.toml

```diff
@@ -13,6 +13,12 @@
   UI_LOGIN_STRATEGY_NAMES = ""
   TASKCLUSTER_ROOT_URL = "https://taskcluster.net"
 
+# Rule for redirecting master branch to the heroku app
+[[redirects]]
+  from = "https://taskcluster-web.netlify.com/*"
```

Comment:
@helfi92 I don't think we can pass a URL as the `from` value, it has to be a relative path. I could be wrong though.